tello_driver/src/tello_driver_node.cpp

```diff
@@ -8,24 +8,16 @@ namespace tello_driver
 {
 
 #define TELLO_DRIVER_ALL_PARAMS \
-  CXT_MACRO_MEMBER(               /* Send commands to this IP address */ \
-  drone_ip, \
-  std::string, std::string("192.168.10.1")) \
-  CXT_MACRO_MEMBER(               /* Send commands to this port */ \
-  drone_port, \
-  int, 8889) \
-  CXT_MACRO_MEMBER(               /* Send commands from this port */ \
-  command_port, \
-  int, 38065) \
-  CXT_MACRO_MEMBER(               /* Flight data will arrive at this port */ \
-  data_port, \
-  int, 8890) \
-  CXT_MACRO_MEMBER(               /* Video data will arrive at this port */ \
-  video_port, \
-  int, 11111) \
-  CXT_MACRO_MEMBER(               /* Camera calibration path */ \
-  camera_info_path, \
-  std::string, "install/tello_driver/share/tello_driver/cfg/camera_info.yaml") \
+  CXT_MACRO_MEMBER(image_raw_frame_id, std::string, std::string("camera_frame")) /* Frame for the published image */ \
+  CXT_MACRO_MEMBER(drone_ip, std::string, std::string("192.168.10.1")) /* Send commands to this IP address */ \
+  CXT_MACRO_MEMBER(drone_port, int, 8889)                 /* Send commands to this port */ \
+  CXT_MACRO_MEMBER(command_port, int, 38065)              /* Send commands from this port */ \
+  CXT_MACRO_MEMBER(data_port, int, 8890)                  /* Flight data will arrive at this port */ \
+  CXT_MACRO_MEMBER(video_port, int, 11111)                /* Video data will arrive at this port */ \
+  CXT_MACRO_MEMBER(camera_info_path, std::string, \
+    "install/tello_driver/share/tello_driver/cfg/camera_info.yaml") /* Camera calibration path */ \
+  CXT_MACRO_MEMBER(pub_image_raw_best_effort_not_reliable, int, 0)  /* publish image_raw message with best_effort (gazebo camera, tello_driver)  */ \
+  CXT_MACRO_MEMBER(pub_camera_info_best_effort_not_reliable, int, 1)/* publish camera_info message with best_effort (gazebo camera, tello_driver)  */ \
```

Comment:
Why int instead of bool?

Reply:
That is a good question. When I first created the macros, I think the ROS CL parser wouldn't parse true and false. Maybe it does now.

Reply:
I use bool in a bunch of places.